docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -72,12 +72,20 @@ See the [variables file](https://github.com/2i2c-org/infrastructure/blob/HEAD/te
 Names for Azure container registries and storage accounts **must** conform to the following guidelines:
 
 - alphanumeric strings between 5 and 50 characters for [container registries](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsoftcontainerregistry), e.g., `myContainerRegistry007`
-- alphanumeric strings between 2 and 24 characters for [storage accounts](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsoftstorage), e.g., `myStorageAccount314`
+- lowercase letters and numbers strings between 2 and 24 characters for [storage accounts](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsoftstorage), e.g., `mystorageaccount314`
 
-We explictly recommend the following conventions using `lowerCamelCase`:
+````{ note}
+A failure will occur if you try to create a storage account whose name is not entirely lowercase.
+````
+
+We explictly recommend the following conventions using `lowercamelcase`:
```

Comment:
```suggestion
We recommend the following conventions using `lowercase`:
```

I believe that "lower camel case" means something like `lowerCamelCase` (as opposed to "camel case" which is `CamelCase`). So we are just doing plain old `lowercase`